aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,477 @@
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
+        .AddNegotiate();
+    // Other service configuration removed.
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseAuthentication();
+    // Other app configuration removed.
+}
+```
+
+Host requirements:
+
+* Windows hosts must have [Service Principal Names](/windows/win32/ad/service-principal-names) (SPNs) added to the user account hosting the app.
+* Linux and macOS machines must be joined to the domain.
+
+  * SPNs must be created for the web process.
+  * [Keytab files](https://blogs.technet.microsoft.com/pie/2018/01/03/all-you-need-to-know-about-keytab-files/) must be generated and configured on the host machine.
+
+See <xref:security/authentication/windowsauth> for more information.
+
+## Template changes
+
+The web UI templates (Razor Pages, MVC with controller and views) have the following removed:
+
+* The cookie consent UI is no longer included. To enable the cookie consent feature in an ASP.NET Core 3.0 template generated app, see <xref:security/gdpr>.
+* Scripts and related static assets are now referenced as local files instead of using CDNs. For more information, see [this GitHub issue](https://github.com/aspnet/AspNetCore.Docs/issues/14350).
+
+The Angular template updated to use Angular 8.
+
+## Razor components
+
+Razor components are self-contained chunks of user interface (UI), such as a page, dialog, or form. Razor components are normal .NET classes that define UI rendering logic and client-side event handlers. You can create rich interactive web apps without JavaScript. 
+
+Razor components are typically authored using Razor syntax, a natural blend of HTML and C#. Razor components are similar to Razor Pages and MVC views in that they both use Razor. Unlike pages and views, which are based on a request-response model, components are used specifically for handling UI composition. 
+
+For more information, see <xref:blazor/index>.
+
+## SignalR
+
+See [Update SignalR code](xref:migration/22-to-30#update-signalr-code) for migration instructions.
```

Comment:
I might suggest a small blurb about the move to System.Text.Json (since it's probable some SignalR customers would hit issues). I know there's a reference to it above, but this would call out the  I can't do a suggestion because the suggestion would include code fences, but you could edit this comment, copy out the below and paste it in to "apply" this suggestion.

Another option would be to put this blurb in a new heading section of https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/signalr/configuration.md and then link to it here.

---

SignalR now uses System.Text.Json to implement the JSON Hub Protocol. There are some serialization scenarios previously supported by Newtonsoft.Json which are not currently supported in System.Text.Json. If your server app is affected by one of those scenarios, you can revert to the Newtonsoft.Json-based protocol package by adding a reference to the `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson` NuGet package and chaining a call to `AddNewtonsoftJson()` to your `AddSignalR` call in `ConfigureServices`:

```csharp
services.AddSignalR()
    .AddNewtonsoftJsonProtocol() // Revert to Newtonsoft.Json protocol implementation
```

Similarly, to revert the .NET client to the Newtonsoft.Json-based protocol implementation, add a reference to the `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson` NuGet package and chaining a call to `AddNewtonsoftJson()` to your `HubConnectionBuilder` calls:

```csharp
var connection = new HubConnectionBuilder()
    .WithUrl("...")
    .AddNewtonsoftJsonProtocol() // Revert to Newtonsoft.Json protocol implementation
    .Build();
```

Reply:
@anurse  that's in the migration doc. I did a a pointer to that specifically.